README.md

```diff
@@ -31,6 +31,97 @@ GetText/DecorateFunctionMessage:
   Enabled: true
 ```
 
+## Cops
```

Comment:
Throughout the README, I'd suggest a line space between headers, as well as between text and code samples. GitHub renders OK, but I think it makes things more readable in the raw format.